content/topics/tilde/understanding-the-review-column/_index.md

```diff
@@ -77,4 +77,11 @@ In terms of accuracy, if you mark something as Competent and then a staff member
 
 We dont expect you to be perfect all the time. But we do expect you to keep an eye on cards you've reviewed. If a staff member disagrees with your review you need to look at it and figure out why they disagree with you. Then the next time you review someone's work you need to try to use your new knowledge.
 
-By reviewing code and stating your opinion you will constantly test and upgrade your skills.
\ No newline at end of file
+By reviewing code and stating your opinion you will constantly test and upgrade your skills.
+
+## Instruction:
+
+- You need to demonstrate your understanding of the review column before putting this topic card into the review column. 
+- To do this, FIRST, you need to do one of the project cards, The *How Repo Project work*. Once you've merged it and moved it to the review column you can then move this topic card to the review column for a review.
```

Comment:
I think we should be more specific here, they should only put the Understanding the Review Column topic in review, once these two projects have been marked competent
- Tilde project tutorial: How to submit a link
- Tilde project tutorial: How Repo projects work 

People usually mess up on the second one since this is where PR's need to be merged first.  Anyway having explicit criteria means there's no ambiguity.